repoarchivetool/storage_interface.py

```diff
@@ -76,12 +81,14 @@ def get_bucket_content(bucket: str, filename: str) -> bool | ClientError:
 
 
 def update_bucket_content(bucket: str, filename: str, local_filename: str = "") -> bool | ClientError:
-    """Uploads a given file to an S3 Bucket
+    """Uploads a given file to an S3 Bucket.
 
     ==========
 
     Args:
         filename (str): The name of the file to upload
+        bucket (str): The name of the bucket
```

Comment:
Can the argument docs match the order they are passed in to the function pleasde.